content/en/Getting started/local.md

```diff
@@ -40,55 +47,276 @@ There are two steps in getting Opta Local running. First, Opta will create a Kub
 
 Running services in Opta Local environment is almost identical to how you would run them on public cloud using the Opta CLI. We show a couple of opta yaml examples to get you started.
 
-#### Example 1: Stateless Service
+#### Example 1: Deploy a service using an existing docker image
```

Comment:
So local doesn't need the env file. We should be able to simplify this right? by not mentioning the local.yaml

Reply:
I was also thinking that way at first, but if we want to demo `opta deploy` (as used in example 2) we can't only use one file: it will throw an error that deploy doesn't work on environment file.
Let me see if there is a way to use one file for example 1 and introduce the env file for example 2.

Reply:
cc @bigbitbus. So from what I remember you should still be able to use just one file. what opta local does is kinda of generates a simple env file for local automagically. (i could be wrong)

Reply:
here is the single file I tried:
```yaml
# opta.yaml
name: hello
environments:
  - name: local
    path: "opta.yaml"
modules:
  - type: k8s-service
    name: hello
    port:
      http: 80
    image: ghcr.io/run-x/opta-examples/hello-app:main
    healthcheck_path: "/"
    public_uri: "/hello"
```
and the error I got when using `deploy`
```
opta deploy --auto-approve -c opta.yaml --image hello-app:v2 
ERROR: Environment hello can not have an environment itself (usually this means your file is self-referencing as it's own parent).
If you need more help please reach out to the contributors in our slack channel at: https://slack.opta.dev
```

Reply:
can you remove the environments section from this file. and try again.

Reply:
try #1
```
name: hello
modules:
  - type: k8s-service
    name: hello
    port:
      http: 80
    # from https://github.com/run-x/opta-examples/tree/main/hello-app
    image: ghcr.io/run-x/opta-examples/hello-app:main
    healthcheck_path: "/"
    public_uri: "/hello"
```
```
opta apply --local --auto-approve -c opta-modified.yaml 
ERROR: Config must have org name or a parent who has an org name
```

try #2
```
name: hello
org_name: my-org
modules:
  - type: k8s-service
    name: hello
    port:
      http: 80
    # from https://github.com/run-x/opta-examples/tree/main/hello-app
    image: ghcr.io/run-x/opta-examples/hello-app:main
    healthcheck_path: "/"
    public_uri: "/hello"

```
```
opta apply --local --auto-approve -c opta-modified2.yaml
ERROR: No cloud provider (AWS, GCP, or Azure) found
```

Reply:
I updated the instructions to use 1 file for example 1 and 2 files for example 2.
It makes example 1 simplier, and I added a step to delete the kind cluster in example 2.

I found a few bugs when running `opta destroy`, I created some jiras so we can address them.

Reply:
hmm yeah. Ok. So this needs some parent... We should rethink this restriction at some point. but this is fine for now.